src/SFML/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if(CSFML_LINK_SFML_STATICALLY)
     set(SFML_STATIC_LIBRARIES TRUE)
     add_definitions(-DSFML_STATIC)
 endif()
-find_package(SFML 2 COMPONENTS network graphics audio REQUIRED)
+find_package(SFML 2.5.1 COMPONENTS network graphics audio REQUIRED)
```

Comment:
I guess we should be specific, since we expect stuff to be around, but I feel like it should be specific up to minor version and not patch version.

Reply:
Do we want to allow for someone using 2.5.0?

Reply:
Why not? It's their decision to make 😄

Reply:
You tell me where we're drawing the line. If 2.5.0 is the lowest we're allowing then so be it. Do we instead allow 2.4.0? Or 2.1? That's up to you. Just make a decision.

Reply:
Yeah, 2.0.0 is wrong, because CSFML isn't compatible with SFML 2.0.0 anymore, since we do use the newer APIs, but the compatibility is on the patch version. We don't "care" if they use 2.5.0, 2.5.1 or a hypothetical 2.5.2 🙂